assets/javascript/logic.js

```diff
@@ -64,18 +64,21 @@ function zomatoGetRestaurants(a) {
         success: function (data) {
             console.log(data);
             var z = data.restaurants;
+            var labels = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
+            var labelIndex = 0;
             $("#table-head").empty();
```

Comment:
I've already started on the js file. Did you do a pull .. ?

Reply:
I submitted a PR about 2 hours ago.

Reply:
Once it is approved and merged you can pull it to your branch and it should not affect anything that you have been working on.

Reply:
I also do not see a branch for you on the repository so make sure you are working on your own branch